src/ctl/ctl.c

```diff
@@ -81,32 +64,41 @@ char *Strdup(const char *s) {
     return p;
 }
 
-umf_result_t umfCtlGet(const char *name, void *ctx, void *arg) {
-    if (name == NULL || arg == NULL || ctx == NULL) {
+umf_result_t umfCtlGet(const char *name, void *ctx, void *arg, size_t size) {
+    if (name == NULL || arg == NULL) {
+        return UMF_RESULT_ERROR_INVALID_ARGUMENT;
+    }
+    // using ctx is disallowed for default settings
```

Comment:
There should not be any assumptions about ctl tree in ctl.c file

Reply:
I moved UmfCtl* functions to `libumf.c`